pandas/core/groupby/groupby.py

```diff
@@ -3089,6 +3139,7 @@ def min(
         min_count: int = -1,
         engine: Literal["cython", "numba"] | None = None,
         engine_kwargs: dict[str, bool] | None = None,
+        **kwargs,
```

Comment:
I think we should try very hard to avoid adding kwargs to a method for internal use.